lib/node_modules/@stdlib/math/base/special/truncn/README.md

```diff
@@ -89,6 +89,92 @@ for ( i = 0; i < 100; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/special/truncn.h"
+```
+
+#### stdlib_base_truncn( x, n )
+
+Rounds a `numeric` value to the nearest multiple of `10^n` toward zero.
+
+```c
+double y = stdlib_base_truncn( 3.141592653589793, -4 );
+// 3.1415
+```
+
+The function accepts the following arguments:
+
+-   **x**: `[in] double` input value.
+-   **n**: `[in] int32_t` integer power of 10.
+
+```c
+double stdlib_base_truncn( const double x, const int32_t n );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/math/base/special/truncn.h"
+#include <stdio.h>
+
+int main( void ) {
+    const double x[] = { 3.14, -3.14, 0.0, 0.0 / 0.0 };
```

Comment:
@gunjjoshi I suggest to update the example code values here and in the `example.c` file to actually perform rounding.